buildkite/scripts/debian/install.sh

```diff
@@ -29,12 +29,12 @@ source ./buildkite/scripts/export-git-env-vars.sh
 # Download required debians from bucket locally
 if [ -z "$DEBS" ]; then 
     echo "DEBS env var is empty. It should contains comma delimitered names of debians to install"
-    exit -1
+    exit 1
 else
   debs=(${DEBS//,/ })
   for i in "${debs[@]}"; do
     case $i in
-      mina-berkeley|mina-devnet|mina-mainnet|mina-berkeley-lightnet)
+      mina-berkeley*|mina-devnet|mina-mainnet)
```

Comment:
>I suggest replacing -1 with 1 in a separate mini-PR.

Sure

> As for this line, aren't we going to remove berkeley in its entirety soon? Is this change required to run the new CI job?

Sorry for this obfuscated line! i wanted to measure test coverage in this test so i need to used mina-berkeley-instrumented so instead of adding mina-berkeley-instrumented case here i just used regex.

Berkeley will be dropped soon, but not so soon judging from amount of code that need to be reviewed.